test/Libraries/WorkflowTests/GeometryDefectTests.cs

```diff
@@ -642,8 +642,8 @@ public void TestNodeChange_Surface_ByJoin()
             RunModel(openPath);
 
             AssertNoDummyNodes();
-            var val = GetPreviewValue("6fd810e6-45a3-43df-9303-8a0a6ce59e45") as Surface;
-            var val2 = GetPreviewValue("39df02d4-8b76-4a00-ac05-c0b7beef4db9") as Surface;
+            var val = GetPreviewValue("ac351e9f-01a4-42b3-9b49-6d764d0224b8") as Surface;
```

Comment:
What were the changes made to this DYN file? Did you remove the Surface.Join node from the graph?

Reply:
Yes removed the surface.join node and added Polysurface.ByjoinedSurfaces node.